sdk/lib/interfaces/Origin.ts

```diff
@@ -80,6 +81,8 @@ export class Origin<T extends Host> {
         addressInfo,
         type,
         masked,
+        hostKind: "multi",
+        hostnames: [],
```

Comment:
shouldn't be necessary - can be looked up from addressInfo.hostId

Reply:
Cleaned up